### PR TITLE
Organization API Endpoint

### PR DESCRIPTION
**Pull request by Jack-Lawton**
Code for an organization endpoint.

Looks good to me, but I am new to Django!

### CODE REVIEW

File: coalesce/organizations/test/test_views.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class TestOrganizationCreate(APITestCase):
+    """
+    Tests /organizers create operations.
+    """
+
+    def setUp(self):
+        self.organization = OrganizationFactory()
+        self.client.credentials(HTTP_AUTHORIZATION=f'Token {self.user.auth_token}')
```
Comment (stefanha) [inline coalesce/organizations/test/test_views.py:16]: This test does not pass with `docker-compose exec api ./manage.py test`:
```AttributeError: 'TestOrganizationCreate' object has no attribute 'user```

Adding a User should solve that:
```python
self.user = UserFactory()```

The `test_post_request_with_valid_data_succeeds()` test fails because HTTP 400 Bad Request is returned instead of 201. I think something is still missing to get the test passing.

File: coalesce/volunteers/migrations/0002_auto_20210420_1138.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    dependencies = [
+        ('volunteers', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='volunteer',
+            name='organizer_comments',
+            field=models.TextField(help_text='Notes from organizers about this volunteer'),
+        ),
```
Comment (stefanha) [inline coalesce/volunteers/migrations/0002_auto_20210420_1138.py:17]: This change to volunteers.models.Volunteer was not made in your pull request. Looks like this migration was accidentally forgotten in a previous pull request. It can be included here or as a separate commit.

File: coalesce/organizations/test/test_views.py (hunk 1 above)
Comment (Jack-Lawton) [reply coalesce/organizations/test/test_views.py]: This should be fixed now

File: coalesce/volunteers/migrations/0002_auto_20210420_1138.py (hunk 2 above)
Comment (Jack-Lawton) [reply coalesce/volunteers/migrations/0002_auto_20210420_1138.py:17]: Happy to leave it here or revert, your call

**Comment by stefanha** [comment]
Thank you! The code has been merged.

I rebased and squashed the intermediate commits, converting Windows line endings to UNIX line endings in new files. I moved the Volunteer organizer_comments migration that Django detected into its own commit since it is not related to the Organization API Endpoint.
